forgit.plugin.zsh

```diff
@@ -182,6 +182,21 @@ forgit::rebase() {
         xargs -I% git rebase -i %
 }
 
+forgit::checkout() {
+    forgit::inside_work_tree || return 1
+    local cmd preview opts files
+    [[ $FORGIT_LOG_GRAPH_ENABLE == false ]] && graph=
+    cmd="git branch --color=always --all --verbose $* $forgit_emojify"
+    preview="echo {} | cut -c 32-39 | xargs -I% git log --color=always --name-only --max-count=1 % | $forgit_show_pager"
```

Comment:
Hi @b10n1k ,
`32-39` seems to be a hard coded range. I tested this command in this repo and got preview errors: `fatal: ambiguous argument '877d6b d': unknown revision or path not in the working tree.` (And I noticed remote branches does not display a commit hash.)
How about replacing it with the following?
```
preview="echo {} | sed 's/^*/ /' | awk '{print \$1}' | xargs -I% git log --color=always --name-only --max-count=1 % | $forgit_show_pager"
```

Reply:
sure. look neat in first glance. let me try and i will let you know

Reply:
@wfxr i encountered issues and then i took a look in the suggestion from @NullSense. This changed a bit also the preview. Can you test it and let me know?

Reply:
@b10n1k Sorry for the late reply. I tested the new implementation. It looks good to me. Just one little thing, would you mind changing the log format to be consistent with `forgit::log`? Or we might add an option for the log format but for now I think it was a bit over-designed. What's your opinion on this?

Reply:
@wfxr i agree with you. ihave already made the change (here)[2548341]